.circleci/config.yml

```diff
@@ -255,6 +255,115 @@ jobs:
             MOCHA_FILE: integration-tests/test-results/junit/test-results-[hash].xml
           when: always
       - upload_artifacts
+  accessibility_test:
+    resource_class: medium+
+    docker:
+      - image: cimg/base:stable-20.04
+        auth:
+          username: dockstoretestuser
+          password: $DOCKERHUB_PASSWORD
+      - image: cimg/postgres:13.3
+        command: postgres -c max_connections=200 -c jit=off
+        auth:
+          username: dockstoretestuser
+          password: $DOCKERHUB_PASSWORD
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+      - image: docker.elastic.co/elasticsearch/elasticsearch:7.10.1
```

Comment:
ditto for these various sessions, wonder if you could even re-use sections (can be a new ticket)

Reply:
I did create an executor for the accessibility tests, however, a better solution probably exists. So, I created this ticket.

[SEAB-4703](https://ucsc-cgl.atlassian.net/browse/SEAB-4703)